src/Inkluzitron/Modules/UserModule.cs

```diff
@@ -291,8 +347,9 @@ public async Task ChangeConsentAsync(PermissionAction action, ConsentType consen
         {
             var dbConsent = consent switch
             {
-                ConsentType.All => CommandConsent.BdsmImageCommands,
+                ConsentType.All => CommandConsent.BdsmImageCommands | CommandConsent.ShowBadges,
```

Comment:
Tento způsob přidání dalšího typu souhlasu ale znamená, že všichni, kdo teď mají "all", tak budou mít odznaky vyplé, pokud tomu správně rozumím.
Pokud to není očekávané chování, možná by stálo za to přidat vyloženě speciální "all" hodnotu (třeba samé 1, aby to dobře hrálo s bit flagy), která by byla dopředně kompatibilní.

Reply:
Yep, předělal jsem to, takže teď to funguje tak jak popisuješ